rerun_cpp/src/rerun/archetypes/boxes3d_ext.cpp

```diff
@@ -32,15 +32,15 @@ namespace rerun {
         ///
         /// TODO(#3285): Does *not* preserve data as-is and instead creates half-sizes from the
         /// input data.
-        /// TODO(#3794): This should not take an std::vector.
+        /// TODO(andreas): This should not take an std::vector.
```

Comment:
1. This not just this file but all of the extensions.
2. I don't want/don't have time to be playing with fire right now :grimacing:

Reply:
@Wumpf

Reply:
let's leave it be. It affects a handful of extensions but not _that_ many. (e.g. just a few lines above things are `Collection` as they should be)